pkg/gatherers/clusterconfig/container_images.go

```diff
@@ -202,3 +205,18 @@ func gatherImages(startMonth string, img2month2count img2Month2CountMap, contain
 		}
 	}
 }
+
+// obfuscateContainerEnvVars finds env variables within the given container list
+// and, if they are a target, it will obfuscate their value
+func obfuscateContainerEnvVars(containers []corev1.Container) {
+	targets := []string{"HTTP_PROXY", "HTTPS_PROXY"}
+	search := regexp.MustCompile(strings.Join(targets, "|"))
+
+	for i := range containers {
+		for j := range containers[i].Env {
+			if search.MatchString(containers[i].Env[j].Name) {
+				containers[i].Env[j].Value = "<obfuscated>"
```

Comment:
Nit again: We usually repeat `x` (for the value) in this case of "default" obfuscation. See https://github.com/openshift/insights-operator/blob/master/pkg/utils/anonymize/string.go.